packages/daemon/bootstrap.js

```diff
@@ -41,6 +42,9 @@ function createPidfileSync(pathname) {
   // create pidfile firstly
   createPidfileSync(DAEMON_PIDFILE);
 
+  // run migration in sub-process
+  exec("npm run migration");
```

Comment:
I think the down script should be executed if any error happens when migration.

Reply:
If any error occurs during migration, no changes will be committed to the database. Therefore, in this case, I do not think we need to run the `down script`.